src/ext/java/org/opentripplanner/ext/fares/impl/HSLFareServiceFactory.java

```diff
@@ -74,6 +78,12 @@ protected void fillFareRules(
   public FareService makeFareService() {
     HSLFareServiceImpl fareService = new HSLFareServiceImpl();
     fareService.addFareRules(FareType.regular, regularFareRules.values());
+    for (FareRuleSet ruleSet : regularFareRules.values()) {
+      LOG.debug("farerule {}", ruleSet);
+      LOG.debug("ruleattr {}", ruleSet.getFareAttribute());
+      LOG.debug("hasAgency {}", ruleSet.hasAgencyDefined());
+      LOG.debug("agency {}", ruleSet.getAgency());
+    }
```

Comment:
```suggestion
    if(LOG.isDebugEnabled()) {
    for (FareRuleSet ruleSet : regularFareRules.values()) {
      LOG.debug("farerule {}", ruleSet);
      LOG.debug("ruleattr {}", ruleSet.getFareAttribute());
      LOG.debug("hasAgency {}", ruleSet.hasAgencyDefined());
      LOG.debug("agency {}", ruleSet.getAgency());
    }
    }
```

Logging inside a loop can have performance impacts even when not used so it's best to check before iterating.